addons/binding/pom.xml

```diff
@@ -59,6 +59,7 @@
     <module>org.openhab.binding.network.test</module>
     <module>org.openhab.binding.nest</module>
     <module>org.openhab.binding.nest.test</module>
+    <module>org.openhab.binding.nibeuplink</module>
```

Comment:
You also need to add the binding to the `feature.xml`.

Reply:
Is there any documentation what exactly needs to be added? I can copy from another entry (in fact I did this for now) but I can see that in some cases dependencies are defined and in some cases multiple features are defined.

As we are already talking about dependencies: Some users experience the error that dependencies (jackson stuff for json processing) that are defined in 'MANIFEST.MF' are not installed. See here: https://community.openhab.org/t/new-binding-available-solaredge-binding/34852/14 I do not have any idea about this especially as this does not occour on my machine. Maybe in my setup jackson is already enforced by another binding...

Reply:
I'm also not sure what to add? From what I found in the feature.xml file it looks like it's needed for specific protocols, like serial.

Talking about the dependencies. In eclipse you can see where what jackson jar it uses to link to, maybe that gives a hint. Also the other bindings using jackson all seem to have added the jackson jar in the lib map.

Reply:
I checked other bindings and all of them have a lib folder containing the jackson binary jars which is checked in in git. I thought this approach is considered bad practice nowadays where people are considered to use dependency management at compile time (e.g. maven, gradle) or at runtime (OSGI)

Reply:
I'm not sure about the rest of the context, but for JSON you should use GSON, this is provided by default by eclipse Smarthome and no additional things have to be done. 

There might be existing bindings which package jackson but the prefered way is to use GSON as most bindings do.

Reply:
@martinvw thanks for the hint. I have replaced jackson with gson which took me only 5 minutes. Now the dependency problem seems to be solved as gson is provided by the platform. Thanks for your help!